raidprotect/src/main.rs

```diff
@@ -55,4 +55,18 @@ mod translations {
     //! This module contains translations generated with [`rosetta-i18n`].
 
     rosetta_i18n::include_translations!();
+
+    /// Returns the desired language from the given argument.
+    /// Default language is english
+    /// Reffer to <https://discord.com/developers/docs/reference#locales> for locale names
+    /// Ex: 'fr' gives you the french language
```

Comment:
The documentation should be on the module to avoid overwriting trait docs.

```suggestion
    //!
    //! [ `From<&str>`] is implemented to get the `Lang` corresponding to a
    //! locale code. See [Discord Docs/Locales].
    //!
    //! [Discord Docs/Locales]: https://discord.com/developers/docs/reference#locales

    rosetta_i18n::include_translations!();
    
```